websites/R/Reaper Scans/presence.ts

```diff
@@ -3,87 +3,78 @@ const presence = new Presence({
-		presenceData.details = "Viewing Comic List";
-		presenceData.state = `📋 ${comics.toString()} comics found`;
-	} else if (/^\/all-series\/novels+\/?$/.test(pathname)) {
-		// Counting novels
-		comics = document.querySelectorAll(".page-listing-item .row .col-6").length;
-		presenceData.details = "Viewing Novel List";
-		presenceData.state = `📋 ${comics.toString()} novels found`;
-	} else if (/^\/all-series\/comics\/manhwas\/?$/.test(pathname)) {
-		// Counting manhwa
-		comics = document.querySelectorAll(".page-listing-item .row .col-6").length;
-		presenceData.details = "Viewing Manhwa List";
-		presenceData.state = `📋 ${comics.toString()} manhwa found`;
-	} else if (/^\/all-series\/comics\/manhuas\/?$/.test(pathname)) {
-		// Counting manhua
-		comics = document.querySelectorAll(".page-listing-item .row .col-4").length;
-		presenceData.details = "Viewing Manhua List";
-		presenceData.state = `📋 ${comics.toString()} manhua found`;
-	} else if (/^\/series\/[0-9a-z-]+\/?$/i.test(pathname)) {
-		presenceData.details = "Viewing Comic";
-		presenceData.state = document.querySelector(".post-title h1").textContent;
-		presenceData.smallImageKey = "view";
-		presenceData.buttons = [
-			{
-				label: "Visit Comic Page",
-				url: origin + pathname,
-			},
-		];
-	} else if (
-		/^\/series\/[0-9a-z-]+\/+(chapter|ch)-[0-9]+\/?$/i.test(pathname)
-	) {
-		let progress =
-			(document.documentElement.scrollTop /
-				(document.querySelector(".read-container").scrollHeight -
-					window.innerHeight)) *
-			100;
-		progress = Math.ceil(progress) > 100 ? 100 : Math.ceil(progress);
+	if ((pathSplit[0] ?? "") === "") presenceData.details = "Viewing Home Page";
+	else if (pathSplit[0] === "comics" || pathSplit[0] === "novels") {
+		const captitalized = `${pathSplit[0][0].toUpperCase()}${pathSplit[0].slice(
+			1
+		)}`;
+		if (pathSplit[1]) {
+			if (pathSplit[2] === "chapters") {
+				let progress =
+					(document.documentElement.scrollTop /
+						(document.querySelector(
+							pathSplit[0] === "comics"
+								? "main > div:not([id*='Ads'])"
+								: "article"
+						).scrollHeight -
+							window.innerHeight)) *
+					100;
+				progress = Math.ceil(progress) > 100 ? 100 : Math.ceil(progress);
 
-		presenceData.details = document.querySelector(
-			"ol.breadcrumb li:nth-child(3)"
-		).textContent;
-		presenceData.state = `📖 ${
-			document.querySelector("ol.breadcrumb li:nth-child(4)").textContent
-		} 🔸 ${progress}%`;
-		presenceData.smallImageKey = "read";
-		presenceData.buttons = [
-			{
-				label: "Visit Comic Page",
-				url:
-					origin +
-					document.querySelector<HTMLAnchorElement>(
-						"ol.breadcrumb li:nth-child(3) a"
-					).href,
-			},
-			{
-				label: "Visit Chapter",
-				url: origin + pathname,
-			},
-		];
-	} else {
+				presenceData.details = `Reading a ${pathSplit[0]}`;
+				presenceData.state = `📖 ${
+					document.querySelector("h2").textContent
+				} ${document.querySelector("h1").textContent.trim()} 🔸 ${progress}%`;
```

Comment:
This is kinda meh.

![](https://user-images.githubusercontent.com/32113157/194187255-248c4eeb-4fbd-4be2-ae7b-b9ec077eca07.png)

Show only the title and chapter/progress just like other manga/manhwa presence.

e.g.
```
Secret Player
Chapter 13 • 2
```

https://github.com/PreMiD/Presences/blob/24563ceb071ae35f2ff99eb681e7aee84479b87f/websites/M/MangaKakalot/presence.ts#L19-L20